scripts/ci/docker-compose/devcontainer.env

```diff
@@ -18,6 +18,7 @@ HOME=
 AIRFLOW_CI_IMAGE="ghcr.io/apache/airflow/main/ci/python3.8:latest"
 ANSWER=
 AIRFLOW_ENABLE_AIP_44="true"
+AIRFLOW_ENV="development"
```

Comment:
Hi @potiuk , I'm not sure where I can set this environment variable so that once breeze starts, it's set as "development"

Reply:
If you want to set it always:

1) Dockerfile.ci (requires rebuilding the image) or
2) entrypoint_ci.sh at the beginning

But I do not think you want to hard-code it and alwaays have it in in  breeze. I thnk breeze airflow should be "as standard as possible" and iany dev mode should be set explicitly. 

I think it would be better to only it in case you run `start-airflow` with `--dev-mode` (flag we already have). So maybe rather than inventing a new variable you should use the existing `DEV_MODE` variable for that. We already have it set to `true` when you pass `--dev-mode` either to `shell` command or `start-airflow`

You can search and see where DEV_MODE is set alredy:

  * scripts/ci/docker-compose/_docker.env
  * scripts/ci/docker-compose/base.yml

And the value ("true/false") is set in src/airflow_breeze/utils/docker_command_utils.py in the MAP where you map "dev_mode" attribute into DEV_MODE env variable.

Reply:
Sure, that will work as well.

Then I think what you have now should be fine - looks like you set the AIRFLOW_ENV in all the places that are needed.

Reply:
Not sure what happened. I had been checking this and it wasn't working. Just pruned docker as breeze was complaining of space and as I tried again, the env is now there. Thanks!